csc-overrides/assets/snippets/auth-update-ssh.md

```diff
@@ -0,0 +1,7 @@
+!!! warning "Significant changes to authentication coming in April 2025"
+    Starting April 14 2025, SSH login to Puhti and Mahti will only work using
```

Comment:
In latest message (to be sent today afternoon) this is clarified a bit (keys stored in .ssh folder vs. authorized keys)